frontend/src/components/samples/details/SampleDetailsContent.js

```diff
@@ -328,10 +328,27 @@ const SampleDetailsContent = ({
         </TabPane>
 
         <TabPane tab={`Lineage`} key="lineage" style={tabStyle}>
-          <SampleDetailsLineage sample={sample}/>
+          <SampleDetailsLineage sample={sample} 
+            handleSampleClick={
+              // Navigate to another sample in the lineage graph
+              (sample_id) => {
```

Comment:
I would prefer not to have those function inline. Assign them to const to make it more readable. to reduce brackets you can use the form sample_id && history() i think.